paddle.js

```diff
@@ -0,0 +1,38 @@
+class Paddle {
+  constructor(ctx, width, height, x, y, speed, color) {
+    this.ctx = ctx;
+    this.width = width;
+    this.height = height;
+    this.x = x;
+    this.y = y;
+    this.speed = speed;
+    this.color = color;
+    this.direction = 0;
+  }
+
+  move() {
+    if (this.left === 0) {
```

Comment:
left is not defined in constructor

and what is 0 and 400? if it is canvas boundaries, why not pass in the canvas and do

`if (0 >= this.x && this.x + this.width <= this.canvas.width)`